sklearn/base.py

```diff
@@ -1087,3 +1088,36 @@ def is_outlier_detector(estimator):
         True if estimator is an outlier detector and False otherwise.
     """
     return getattr(estimator, "_estimator_type", None) == "outlier_detector"
+
+
+def _fit_context(*, skip_nested_validation=True):
+    """Decorator to run the fit methods of estimators within a context manager.
+
+    Parameters
+    ----------
+    skip_nested_validation : bool, default=True
+        Whether to skip the validation of parameters of inner estimators or functions
+        called during fit.
```

Comment:
```suggestion
        called during :term:`fit`.
```

Reply:
It's not needed since it's in a private function and thus won't be rendered.